includes/helper/class-wp-job-manager-helper-api.php

```diff
@@ -144,10 +144,24 @@ public function bulk_activate( $license_key, $product_slugs ) {
 	 * @return array|false JSON response or false if failed.
 	 */
 	public function deactivate( $args ) {
-		$args            = wp_parse_args( $args );
-		$args['wc-api']  = 'wp_plugin_licencing_activation_api';
-		$args['request'] = 'deactivate';
-		return $this->request( $args, false );
+		$args     = wp_parse_args( $args );
+		$response = $this->request_endpoint(
+			'wp-json/wpjmcom-licensing/v1/deactivate',
+			[
+				'method' => 'POST',
+				'body'   => wp_json_encode(
+					[
+						'product_slug' => $args['api_product_id'],
+						'license_key'  => $args['license_key'],
+						'site_url'     => $this->get_site_url(),
+					]
+				),
+			]
+		);
+		if ( ! is_array( $response ) || ! array_key_exists( 'success', $response ) ) {
+			return false;
+		}
+		return $response;
```

Comment:
I thought about adding some handling here to return the same format that we have for errors here, like we have on WPJMCOM for maximum compatibility.

However, I decided to just return `false` here whenever `success` isn't present on the `$response` because I think it matches the return value of the method more.